### DISCUSSION TITLE
Is there anyway to use my mac webcam on a VM using UTM?🤔

### DISCUSSION THREAD

**Discussion by nicrob97** [utmapp/UTM#3856] [UTM Help (macOS)]
Hello, i am using UTM on a mac M1.

I did install 2 VM with UTM:
1 windows 10 x64 in emulation mode
1 windows 11 ARM in virtualization mode;

But in none of the 2 I could use the webcam...

So my question is is it possible to use my mac webcam on a windows VM?
Other question is it possible to use my mac webcam on another OS VM (linux or even mac)?

Thanks

**Comment by basicallyar** [comment]
### According to the official UTM Documentation at [docs.getutm.app/guest-support/sharing/usb](url)
NOTE:
Some devices cannot be captured including built-in Apple webcams and (iOS) some external flash drives.
There is absolutely no way to connect you built in Webcam to your VM. Parallels supports Apple built in webcams unlike UTM 
(NOT SPONSORED)

**Comment by merpsoftcom** [comment]
But windows app in mac for taking remote desktop is able to capture inbuilt mac camera.

**Reply by UniqueBlood7899** [reply]
Can you help me set this up?

**Comment by codethief** [comment]
Wouldn't it be possible for UTM (the VM manager, not the VM guest) to access the webcam like any other MacOS app that uses the camera and then present the camera stream to the VM as some virtual device? In other words: Don't pass through the actual hardware device but provide a virtual camera device.
